### ISSUE TITLE
Ability to reset individual `@storybook/addon-controls` controls

### ISSUE THREAD

**Issue by emattias** [storybookjs/storybook#18223]
Right now you can reset all controls by pressing this button:

<img width="112" alt="CleanShot 2022-05-13 at 16 24 20@2x" src="https://user-images.githubusercontent.com/351537/168304769-94ea61af-29a7-49bb-9ccc-ae1104939163.png">

Sometimes I want to reset only one field. Please add that ability

**Comment by seanmaisch** [comment]
Agreed, I would like this as well please, especially when dealing with stories with lots of controls configured and you only want one reset (like a color).

**Comment by lucapollani** [comment]
Agree on this, dealing with this kind of controls like "text" when there's not a default value, there's not actually the possibility to reset them to "undefined", the will always pass empty string at least.

![image](https://github.com/storybookjs/storybook/assets/71693205/e6884c19-aa85-43d7-b7b1-7baa3d7febb4)

PS. Probably need to add the label "addon: controls" to the issue in order to track it better.
